WARNING  pytest_result_log:plugin.py:122 Start: testcase/test07.py::test_sea_file_info[8ac00e05a7652b31598a56d764bd5b5f7c45a6ef-200-name]
WARNING  pytest_result_log:plugin.py:128 End: testcase/test07.py::test_sea_file_info[8ac00e05a7652b31598a56d764bd5b5f7c45a6ef-200-name]